### DISCUSSION TITLE
Leaflet Map Support

### DISCUSSION THREAD

**Discussion by wchorski** [oleeskild/obsidian-digital-garden#556] [Ideas]
Possible to render leaflet UI onto published gd site?

[Leaflet Obsidian Plugin](https://github.com/javalent/obsidian-leaflet)

**Comment by cathelijne** [comment]
This and Mapview would be awesome <3

**Comment by RobertFiorentino** [comment]
+1!

**Comment by nk-fouque** [comment]
+1

**Comment by JClement57** [comment]
+1!!

**Comment by LytTheBit** [comment]
+1
